easybuild/easyblocks/p/python.py

```diff
@@ -128,6 +129,21 @@ def __init__(self, *args, **kwargs):
             easybuild_subdir = log_path()
             self.pythonpath = os.path.join(easybuild_subdir, 'python')
 
+        ext_defaults = {
+            'download_dep_fail': True,
+            # The only installed packages at this point are default installed ones, e.g. pip&setuptools
+            # And we want to upgrade them cleanly, i.e. uninstall them
+            'pip_ignore_installed': False,
+            # Python installations must be clean
+            'sanity_pip_check': True,
+        }
+        # Disable templating to update the underlying values
+        with disable_templating(self.cfg):
+            for key, value in ext_defaults.items():
+                if key not in self.cfg['exts_default_options']:
+                    self.cfg['exts_default_options'][key] = value
+        self.log.info("exts_default_options: %s", self.cfg['exts_default_options'])
```

Comment:
This should be `debug` unless you make it a more meaningful log message imho

Reply:
Copied from PythonBundle, but downgraded as suggested as that makes sense